blog/views/admin_category_views.py

```
@@ -0,0 +1,136 @@
+    )
+
+    return render(
+        request,
+        "blog/admin/category_manage.html",
+        {"categories": categories},
+    )
+
+
+@admin_access_required
+@require_POST
+def admin_category_create_view(request):
```

Ruff (0.15.6)

[warning] Too many return statements (10 > 6)

(PLR0911)